src/ui/redux/actions/settings.js

```diff
@@ -82,3 +83,30 @@ export function doSetDarkTime(value, options) {
     dispatch(doUpdateIsNight());
   };
 }
+
+export function doSetLanguage(language) {
+  return dispatch => {
+    // this should match the behavior/logic in index-web.html
+    fetch('https://lbry.com/i18n/get/lbry-desktop/app-strings/' + language + '.json')
+      .then(r => r.json())
+      .then(j => {
+        window.i18n_messages[language] = j;
```

Comment:
@seanyesmunt is there a technique we can use to force/always trigger a complete re-render when `window.i18n_messages[language]` was not already set?

The only known time we are failing to re-render is when manually refreshing in app-mode.

Reply:
You can set a value in redux here, then pass that value in as a prop to the top level `<App/>` component